### PR TITLE
Added more links to BlueChi blogs and videos

### PR DESCRIPTION
**Pull request by engelmi**
Added more links of BlueChi blogs, newsletter and videos to the readthedocs page.

### CODE REVIEW

**Review by mwperina — APPROVED** [review]
LGTM

**Comment by engelmi** [comment]
Seems like there is an issue in packit with `Librepo error: Cannot download repomd.xml`: 
https://status.packit.dev/issues/2024-09-16-fedora-outage/

And the integration test failed due to a race condition in the test itself which will be fixed in https://github.com/eclipse-bluechi/bluechi/pull/943

Since this PR only affects the readthedocs, lets merge.

File: doc/docs/resources.md [hunk 1]
```diff
@@ -2,9 +2,15 @@
 
 # Additional Resources
```
Comment (mkemel) [inline doc/docs/resources.md:4]: Maybe a link to the workshop as well?

Comment (engelmi) [reply doc/docs/resources.md:4]: Good point! How about a new section `Demos` and linking https://github.com/eclipse-bluechi/bluechi-demos?

Comment (engelmi) [reply doc/docs/resources.md:4]: Updated. 
@mkemel PTAL

Comment (mkemel) [reply doc/docs/resources.md:4]: Great

**Review by mkemel — APPROVED** [review]
LGTM

**Pull request merged**